extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -358,14 +362,23 @@ export class ResourceTypePickerDialog extends DialogBase {
 							)
 						);
 					}
+				} else {
+					toolsNotInstalled.push(tool);
 				}
 			}
 			// Update the informational message
-			this._dialogObject.message = {
-				level: azdata.window.MessageLevel.Information,
-				text: localize('deploymentDialog.InstalledTools', "All required tools are installed now.")
-			};
-			this._dialogObject.okButton.enabled = true;
+			if (toolsNotInstalled.length === 0) {
+				this._dialogObject.message = {
+					level: azdata.window.MessageLevel.Information,
+					text: localize('deploymentDialog.InstalledTools', "All required tools are installed now.")
+				};
+				this._dialogObject.okButton.enabled = true;
+			} else {
+				this._dialogObject.message = {
+					level: azdata.window.MessageLevel.Information,
+					text: localize('deploymentDialog.PendingInstallation', "Following tools: {0} were still not discovered. Please make sure that they are installed, running and discoverable", toolsNotInstalled.map(t => t.displayName).join(','))
+				};
```

Comment:
This sentence could use some work - it doesn't read that well currently. At the very least something like `Unable to discover these tools: {0}...." would be better but I'd run this by PMs too